addons/l10n_ar_website_sale/demo/website_demo.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+
+    <record id="default_website_ri" model="website">
+        <field name="name">(AR) Responsable Inscripto Website</field>
+        <field name="company_id" ref="l10n_ar.company_ri"/>
+        <field name="logo" type="base64" file="l10n_ar_website_sale/static/description/icon.png"/>
+        <field name="domain" model="ir.config_parameter" eval="obj().env['ir.config_parameter'].sudo().get_param('web.base.url')"/>
+    </record>
+    <function model="payment.acquirer" name="copy">
+        <value eval="[ref('payment.payment_acquirer_transfer')]"/>
+        <value eval="{'company_id': ref('l10n_ar.company_ri'), 'state': 'enabled'}"/>
```

Comment:
@zaoral why do we need a copy?  I don't see why you could not use the original one.

Reply:
company_id seems required, so makes sense